README.md

```diff
@@ -1 +1,57 @@
-# Spinner
\ No newline at end of file
+# Spinner
+> Present loading indicators anywhere quickly and easily
+
+A helpful loading spinner tool allowing you to present a UIActivityIndicator view directly in to views and buttons.
+
+## 📦 Installation
+
+### Carthage
+~~~
+github "nodes-ios/Spinner"
+~~~
+
+### CocoaPods
+~~~
+pod 'Spinner', '~> 1.0'
```

Comment:
Did you actually add cocoapods support? Because if we didn't then it should be in the readme :D